src/main/scala/mesosphere/marathon/core/flow/impl/ReviveOffersActor.scala

```diff
@@ -88,6 +92,11 @@ private[impl] class ReviveOffersActor(
     }
   }
 
+  private[this] def suppressOffers(): Unit = {
+    logger.info("=> Suppress offers NOW")
```

Comment:
"NOW"? Are you shouting at the driver? :D

Reply:
lol this is what it was before

Reply:
Yeah, saw that too ^^